Titanium.Web.Proxy/RequestHandler.cs

```diff
@@ -104,61 +103,87 @@ partial class ProxyServer
                         connectRequestHeaders.Add(newHeader);
                     }
 
-                    if (await CheckAuthorization(clientStreamWriter, connectRequestHeaders) == false)
+                    if (httpRemoteUri.Port == 80) 

```

Comment:
HTTPS was not allowed to port 80 before for some reason before. If we don't have any problem now we can remove the check.